meta-mender-qemu/scripts/mender-qemu

```diff
@@ -130,8 +136,8 @@ for maybe_kvm in -enable-kvm ""; do
             -m 256M \
             $BOOTLOADER_ARG \
             -net nic,macaddr="$RANDOM_MAC" \
-            -net user,hostfwd=tcp::8822-:22$QEMU_NET_HOSTFWD \
-            -display vnc=:23 \
+            -net user,hostfwd=tcp::${PORT_NUMBER}-:22$QEMU_NET_HOSTFWD \
+            -display vnc=:${VNC_NUMBER} \
```

Comment:
Q: When have we ever used `vnc`? :p

Reply:
I think I have used it exactly once, probably five years ago! 😂